test/integration/reports/rendering.js

```diff
@@ -11,7 +11,7 @@ module.exports = function LazyTester(target, keys, params) {
     params = _.clone(params || {});
 
     // renders
-    const invalid = { renderer : 'unknown' };
+    const invalid = _.merge({ renderer : 'unknown' }, params);
```

Comment:
Because, I want to make sure, it is testing for an invalid renderer, if I don't specify parameters cash report (which test the existence of parameters at the beginning) will throw an error for missing parameters but not for render.